publish/publish-snapshot.sh

```diff
@@ -111,20 +111,26 @@ fi
 for pom in ${pomFiles}; do
   pom_dir="$(dirname "${pom}")"
   for FILE in "${pom_dir}"/*;  do
-    if [[ ! $FILE == *"-javadoc.jar"* ]] && [[ ! $FILE == *"-sources.jar"* ]]; then
+    if [[ $FILE == $pom ]] ; then
+      continue
+    fi
     extension="${FILE##*.}"
       case $extension in jar|war|zip)
-        echo "Uploading: ${FILE} to ${url}"
+        # Ensure we are only pushing the artifact that ends with -SNAPSHOT.<extension> and its pom.
+        if [[ $FILE == *SNAPSHOT.${extension} ]] ; then
+         echo "Uploading: ${FILE} with ${pom} to ${url}"
          mvn --settings="${mvn_settings}" deploy:deploy-file \
          -DgeneratePom=false \
          -DrepositoryId=nexus \
          -Durl="${SNAPSHOT_REPO_URL}" \
          -DpomFile="${pom}" \
          -Dfile="${FILE}" || echo "Failed to upload ${FILE}"
```

Comment:
Nitpick, spacing needs some improvements right here.

Reply:
Updated - ran shfmt on the file.